packages/esm-patient-forms-app/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "../../tools/setupTests.ts"],
```

Comment:
@denniskigen I did this to allow TS to recognise the types from `@testing-library/jest-dom`. I think we talked about that once before. I think this is safe to do, as we don't currently use tsconfig to for actual compilation, just type-checking. If / when we do, we may need some scheme with a `tsconfig.test.json`, but that seems more complicated.

Reply:
I agree. The existing workaround (needing to manually launch the `test-helpers.tsx`
file to get the IDE to load types) is too convoluted. I've not found a better approach.